meta/Meta.cpp

```diff
@@ -1786,8 +1786,8 @@ sai_status_t Meta::objectTypeGetAvailability(
     PARAMETER_CHECK_OID_OBJECT_TYPE(switchId, SAI_OBJECT_TYPE_SWITCH);
     PARAMETER_CHECK_OID_EXISTS(switchId, SAI_OBJECT_TYPE_SWITCH);
     PARAMETER_CHECK_OBJECT_TYPE_VALID(objectType);
-    PARAMETER_CHECK_POSITIVE(attrCount);
-    PARAMETER_CHECK_IF_NOT_NULL(attrList);
+    //PARAMETER_CHECK_POSITIVE(attrCount);
+    //PARAMETER_CHECK_IF_NOT_NULL(attrList);
```

Comment:
why this is disabled ?

Reply:
Refer to associated changes in sonic-swss/orchagent/crmorch.cpp with https://github.com/Azure/sonic-swss/pull/1686
When checking availability of a resource solely based on OBJECT_TYPE, attrCount is 0 and attrList is NULL.

Reply:
could you add "When checking availability of a resource solely based on OBJECT_TYPE, attrCount is 0 and attrList is NULL." as comment in code ?

Reply:
and remove those 2 lines

Reply:
Done.